pkg/onchain/website/uploader.go

```diff
@@ -55,6 +58,42 @@ func Upload(atAddress string, content string, wallet *wallet.Wallet) (string, er
 	return "1", nil
 }
 
+func Status(websiteCreator string) (*models.UploadState, error) {
+	client := node.NewDefaultClient()
```

Comment:
I'm not sure it's a good idea to set the client here. When we will need to set a specific node IP, a lot of refacto will be needed...

Reply:
To be treated here as we have this usecase in several places of the code

https://github.com/massalabs/thyra/issues/96